deploy/kubernetes/helm/che/custom-charts/che-plugin-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: che-plugin-registry
+  name: che-plugin-registry
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: che-plugin-registry
+  strategy:
+    type: RollingUpdate
+    rollingParams:
```

Comment:
I get `unknown field "rollingParams" in io.k8s.api.extensions.v1beta1.DeploymentStrategy`

It doesn't look like `rollingParams` is in a `DeploymentStrategy` (c.f. [k8s api reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#deploymentstrategy-v1-apps)). That's curious. Have you tested on OpenShift?

Anyway chaging to

```yaml
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
```

worked

Reply:
no, I've tested on minikube. Looks like values were ignored, but I didn't get any error. Fixed

Reply:
I got configs from registries repos (https://github.com/eclipse/che-devfile-registry/blob/master/deploy/kubernetes/che-devfile-registry/templates/deployment.yaml#L24), I'll fix them also.

Reply:
https://github.com/eclipse/che-plugin-registry/pull/187
https://github.com/eclipse/che-devfile-registry/pull/44